projects/netgrif-components-core/src/lib/navigation/breadcrumbs/abstract-breadcrumbs.component.ts

```diff
@@ -38,7 +38,7 @@ export abstract class AbstractBreadcrumbsComponent implements OnDestroy, AfterVi
     private redirectUrls: Map<string, Array<string>>;
     private nicePathSubscription: Subscription;
 
-    protected constructor(protected _uriService: UriService,
+    protected constructor(protected _uriService: PathService,
```

Comment:
Please rename the property to `_pathService` to match the type.

Reply:
Done